pkgs/applications/misc/opentrack/default.nix

```diff
@@ -18,14 +18,14 @@
   makeDesktopItem,
   fetchurl,
 }: let
-  version = "2022.3.0";
+  version = "2023.3.0";
 
   aruco = callPackage ./aruco.nix {};
 
   # license.txt inside the zip file is MIT
   xplaneSdk = fetchzip {
-    url = "https://developer.x-plane.com/wp-content/plugins/code-sample-generation/sample_templates/XPSDK303.zip";
-    sha256 = "11wqjsr996c5qhiv2djsd55gc373a9qcq30dvc6rhzm0fys42zba";
+    url = "https://developer.x-plane.com/wp-content/plugins/code-sample-generation/sample_templates/XPSDK401.zip";
```

Comment:
the url should be
`+    url = "https://developer.x-plane.com/wp-content/plugins/code-sample-generation/sdk_zip_files/XPSDK401.zip";`